pkg/apis/kops/validation/legacy.go

```diff
@@ -117,6 +117,11 @@ func ValidateCluster(c *kops.Cluster, strict bool) *field.Error {
 	if strict && c.Spec.KubeControllerManager == nil {
 		return field.Required(fieldSpec.Child("KubeControllerManager"), "KubeControllerManager not configured")
 	}
+	if kubernetesRelease.LT(semver.MustParse("1.7.0")) {
```

Comment:
nit: 
```go
if kubernetesRelease.LT(semver.MustParse("1.7.0")) && c.Spec.ExternalCloudControllerManager != nil
```

Reply:
please file a PR to move this out of legacy.  validation.go is the new home.